docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -0,0 +1,1183 @@
+ 
+| `SERVICE_MARKET_REDIS`
+| string
+| ✔
+| URL of the cache storage (Redis).
+| redis://localhost:6379/2
+
+| `SERVICE_MARKET_FIAT_CURRENCIES`
+| string
+| ✗
+| Fiat currencies are used for price calculation. 
+All Fiat currencies used here need to be comma separated.
+| EUR,USD,CHF,GBP,RUB 
+
+| `SERVICE_MARKET_TARGET_PAIRS`
+| string
+| ✗
+| Exchange rates exposed to the Gateway.
+The values listed here must be comma separated.
+| LSK_BTC,LSK_EUR,BTC_CHF
+
+| `EXCHANGERATESAPI_IO_API_KEY`
+| string
+| ✔
+| Optional API key for https://exchangeratesapi.io/.
+The `/market/prices` endpoint will respond with additional data, specifically the exchange rates for various cryptocurrencies in other fiat currencies.
+The free plan would suffice for Lisk Service.
+|  
+
+
+|===
+
+--
+Sample::
++
+--
+[source,js]
+----
+module.exports = {
+  apps: [
+    // Other types of configurations.
+    {
+      name: 'lisk-service-market',
+      // [...]
+      env: {
+        // --- Remember to set the properties below
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_MARKET_REDIS: 'redis://localhost:6379/2',
+        SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB',
+        SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF',
+        // EXCHANGERATESAPI_IO_API_KEY: ''
+      },
+    },
+    // Other types of configurations.
+  ],
+};
+----
+--
+=====
+
+=== Export
+
+[tabs]
+=====
+Properties::
++
+--
+[cols="2,1,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Configurable
+|Description
+|Sample
+
+| `SERVICE_BROKER`
+| string
+| ✔
+| URL of the microservice message broker (NATS or Redis).
+| redis://localhost:6379/0
+ 
+| `SERVICE_EXPORT_REDIS`
+| string
+| ✔
+| URL of the permanent cache storage (Redis).
+| redis://localhost:6379/3
+
+| `SERVICE_EXPORT_REDIS_VOLATILE`
+| string
+| ✔
+| URL of the volatile cache storage (Redis).
+| redis://localhost:6379/4
+
+|===
+
+--
+Sample::
++
+--
+[source,js]
+----
+module.exports = {
+  apps: [
+    // Other types of configurations.
+    {
+      name: 'lisk-service-export',
+      // [...]
+      env: {
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_EXPORT_REDIS: 'redis://localhost:6379/3',
+        SERVICE_EXPORT_REDIS_VOLATILE: 'redis://localhost:6379/4',
+      },
+    },
+    // Other types of configurations.
+  ],
+};
+----
+--
+=====
+
+== Lisk Settings
+
+Configurable environment variables related to Lisk node.
+
+=== Node settings
+
+[cols="2,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Description
+|Sample
+
+| `LISK_CORE_WS`
+| string
+| URL for Lisk Core WebSocket RPC API.
+The URL will differ according to the environment i.e., docker, or server installation.
+| ws://localhost:7887 or ws://host.docker.internal:7887
+ 
+| `LISK_CORE_CLIENT_TIMEOUT`
+| string
+| Lisk Core client timeout (in seconds).
+| 30
+|===
+
+
+=== MySQL Snapshot settings for Docker
```

Comment:
Will verify and update later.

Reply:
Will the verification be possible within this PR or should I create a new issue for this for a later time?

Reply:
Let's create a separate PR for this.